templates/partial/piwik.html

```diff
@@ -1,15 +1,15 @@
-<!-- Piwik -->
+<!-- Matomo -->
 <script type="text/javascript">
-  var _paq = _paq || [];
+  var _paq = window._paq || [];
   _paq.push(['trackPageView']);
   _paq.push(['enableLinkTracking']);
   (function() {
-    var u="//{{ PIWIK_URL }}/";
-    _paq.push(['setTrackerUrl', u+'piwik.php']);
-    _paq.push(['setSiteId', {{ PIWIK_SITE_ID }}]);
+    var u="https://{{ MATOMO_URL }}/";
```

Comment:
Changing the name of the config you break everyone using Piwik, it should consider both options.

Example:

`{{ MATOMO_URL or  PIWIK_URL}}`

With this we ensure that people don't need to rewrite the settings, because I believe that the ID's from Piwik still works.